lnrpc/lightning.proto

```diff
@@ -3633,7 +3633,16 @@ message ListInvoiceRequest {
     specified index offset. This can be used to paginate backwards.
     */
     bool reversed = 6;
+
+    // If set, filters out all invoices with a creation date greater than or
```

Comment:
q: shouldn't this be "filters out all invoices with a creation date less than"?

Reply:
make sense. Used "return" instead to avoid confusion.